### PR TITLE
make pagination for About section

### PR DESCRIPTION
**Pull request by amonamoqadi**

### CODE REVIEW

File: src/components/AboutImg/index.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ function AboutImg(props){
         INACTIVE:"Inactive",
         ACTIVE:"Active"
     }
-    var isDisabled = props.status===STATUS.INACTIVE;
+    var isDisabled = props.status === STATUS.INACTIVE;
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:9]: use const

File: src/containers/About/index.js [hunk 2]
```diff
@@ -1,27 +1,63 @@
 import React from 'react';
 import axios from 'axios';
+import ReactPaginate from 'react-paginate';
 import AboutImg from '../../components/AboutImg';
 import janeImage from "../../img/jane.jpg";
 import "./style.scss";
 
-class About extends React.Component{  
+class About extends React.PureComponent{
```
Comment (IbrahimLama) [inline src/containers/About/index.js:8]: what is the advantage of using PureComponent?

File: src/containers/About/index.js [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
-            AboutList:[],
-        };
+        this.state = {   
+            offset: 0,  
+            AboutList: [],
+            orgtableData: [],
+            perPage: 8,
+            currentPage: 0,
+        }
+        this.handlePageClick = this.handlePageClick.bind(this);
```
Comment (IbrahimLama) [inline src/containers/About/index.js:19]: what other ways can you use for binding?

File: src/containers/About/index.js [hunk 4]
```diff
@@ -1,27 +1,63 @@
...
+            this.loadMoreData()
+        });
+    };
+
+    loadMoreData() {
+		const data = this.state.orgtableData;
+        const {offset} = this.state;
+	    const {perPage} = this.state;
+		const slice = data.slice(offset, offset + perPage)
+		this.setState({
```
Comment (IbrahimLama) [inline src/containers/About/index.js:40]: fix the indentation, 
and its better to do it this way 
```
   const {offset, perPage} = this.state;

File: src/containers/About/index.js [hunk 5]
```diff
@@ -1,27 +1,63 @@
...
+        axios.get(`https://gorest.co.in/public-api/users`)
+             .then(res =>
+                {
+                    var data = res.data.data;
+                    var slice = data.slice(this.state.offset, this.state.offset + this.state.perPage);
+                    // console.log(data.length);
+                    this.setState({
+                        pageCount: Math.ceil(data.length / this.state.perPage),
+                        orgtableData: res.data.data,
+                        AboutList: slice
```
Comment (IbrahimLama) [inline src/containers/About/index.js:57]: use const and destructure offset and perpage

File: src/containers/About/index.js (hunk 2 above)
Comment (amonamoqadi) [reply src/containers/About/index.js:8]: it is exactly the same as Component but it handles the shouldComponentUpdate method. I used it at first and I forgot to return it back to Component.

File: src/containers/About/index.js (hunk 3 above)
Comment (IbrahimLama) [reply src/containers/About/index.js:19]: here no need to use bind this way as you are using arrow function in handlePageClick, try to remove this and see if it works. 
```
    handlePageClick = (e) => {

**Comment by amonamoqadi** [comment]
Yes it works

File: src/containers/About/index.js [hunk 6]
```diff
@@ -34,8 +34,7 @@ class About extends React.PureComponent{
 
     loadMoreData() {
 		const data = this.state.orgtableData;
-        const {offset} = this.state;
-	    const {perPage} = this.state;
+        const {offset, perPage} = this.state;
 		const slice = data.slice(offset, offset + perPage)
 		this.setState({
 			pageCount: Math.ceil(data.length / perPage),
```
Comment (IbrahimLama) [inline src/containers/About/index.js]: I don't see the indentation is fixed

**Comment by amonamoqadi** [comment]
It is correct from the first time 3nde .. I don't know why it goes like
this in github!

**Pull request merged**